File: 1794 B. Not Dividing/Solution.py
# 320702149	        May/22/2025 04:56UTC+6	        Krishno_MD	        1794B - Not Dividing	        PyPy 3-64	        Accepted	        264 ms	        9000 KB

# Krishno_Mondol_KUET_CSE
def solution():
    n = int(input())
    v = [x+1 if x==1 else x for x in map(int,input().split())]
    for i in range(1,n):
        if v[i]%v[i-1]==0:
            v[i] += 1
    print(*v)

test = int(input())
for i in range(test):
    solution()

File: 1794 B. Not Dividing/Solution.cpp
// 320702107	        May/22/2025 04:54UTC+6	        Krishno_MD	        1794B - Not Dividing	        C++20 (GCC 13-64)	        Accepted	        124 ms	        100 KB

#include<bits/stdc++.h>
using namespace std;
//Krishno_Mondol_KUET_CSE 
#define ll long long 
void solve(){ 
    int n; cin>>n;
    vector<int> v(n);
    for(int i=0; i<n; i++){cin>>v[i]; if(v[i]==1){v[i] += 1;}}
    for(int i=1; i<n; i++){
        if(v[i]%v[i-1]==0){v[i]++;}
    }
    for(int i=0; i<n; i++){cout<<v[i]<<" ";}
    cout<<endl;
}
int main(){ int t; cin>>t; while(t--){ solve(); } return 0;}
